### PR TITLE
adding a new file on chats

### PR DESCRIPTION
**Pull request by npasha**

### CODE REVIEW

**Comment by npasha** [comment]
this pull request #15

File: India [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#Bangalore
+  -Places to Eat
+    - Koramangala 
+
```
Comment (githubteacher) [inline India:4]: @npasha What would you recommend if I had a few hours after class to do something fun?

**Comment by npasha** [comment]
I would rush to pick up my kid from School but would like to run through seeing what each one of us learnt

**Comment by npasha** [comment]
@harmaildhiman all  varieties of Chats

**Pull request merged**